### ISSUE TITLE
Out-dated data in the repo?

### ISSUE THREAD

**Issue by paulzhn** [googleben/ACNHDialog#12]
The size of `clean.mp` which is downloaded on `https://googleben.com/ACNHDialog/` is different from this repo, and the website's is a little larger. Is the repo not updated?

**Comment by googleben** [comment]
No, I haven't touched this in quite a while and don't really intend to unless someone opens a pull request or such. As for the website, it's actually a front for this repo's github pages, so you can see the source for the website in the gh-pages branch.
